src/tests/java/htsjdk/samtools/DuplicateSetIteratorTest.java

```diff
@@ -56,7 +57,7 @@ public void testSupplementalReads() {
         }
 
         //we expect 15 duplicate sets one for the initial two reads and one for each of the additional 14 reads.
-        assert (allSets.size() == 15);
-        assert (allSets.get("READ0").size() == 2);
+        Assert.assertEquals(allSets.size(), 15, "Wrong number of duplicate sets.");
+        Assert.assertEquals(allSets.get("READ0").size(), 2, "Should be two reads in the READ0 duplicate set, but there are not.");
     }
```

Comment:
![Watching you](http://quotespics.com/wp-content/quote-images/im-watching-you.jpg)